src/pages/dashboard.tsx

```diff
@@ -239,961 +273,1583 @@ const Dashboard = () => {
+                                      </option>
+                                    )
                                   })}
                                 </select>
                               </Col>
                             </div>
                             <div className="row mb-4">
                               <Col sm={12}>
-                                <select className="form-select" onChange={handleDepositRequestTime}>
-                                  <option hidden>Minimum Commitment Period</option>
-                                  {activeDepositsData.filter((asset) => {
-                                    return (EventMap[asset.market.toUpperCase()] === depositRequestSel)
-                                  })
-                                    .map(item => item['commitment'])
-                                    .filter((value, index, self) => self.indexOf(value) === index)
+                                <select
+                                  className="form-select"
+                                  onChange={handleLoanCommitementChange}
+                                >
+                                  <option hidden>
+                                    Minimum Commitment Period
+                                  </option>
+                                  {activeLoansData
+                                    .filter(asset => {
+                                      return (
+                                        EventMap[
+                                          asset.loanMarket.toUpperCase()
+                                        ] === loanOption
+                                      )
+                                    })
+                                    .map(item => item["commitment"])
+                                    .filter(
+                                      (value, index, self) =>
+                                        self.indexOf(value) === index
+                                    )
                                     .map((asset, key) => {
-                                      return <option key={key} value={asset}>{EventMap[asset]}</option>
+                                      return (
+                                        <option key={key} value={asset}>
+                                          {EventMap[asset]}
+                                        </option>
+                                      )
                                     })}
                                 </select>
                               </Col>
                             </div>
+                            <div className="row mb-4">
+                              <Col sm={12}>
+                                <select
+                                  className="form-select"
+                                  onChange={handleCollateralOptionChange}
+                                >
+                                  <option hidden>Collateral Market</option>
```

Comment:
@MPankajArun collateral market needs to be filtered using loanMarket and it's commitment
Currently collateralMarkets of all loans are being displayed (also on production)

Reply:
got it. will be done by next remote push